lib/GitUtils.jsx

```diff
@@ -0,0 +1,44 @@
+const { promisify } = require('util');
+const exec = promisify(require('child_process').exec);
+
+export default class GitUtils {
```

Comment:
Does this work properly? It's really odd to mix the CJS module system with the ES6 module system. You're importing your dependencies using CJS (aka `require()` instead of `import`), and then exporting using ES6 (`export default xxx`  instead of `module.exports = xxx`).

I think this may have only worked because the code is transpiled by `ncc` and bundled with its dependencies before being executed in the Github Actions runner. Since this code is meant to be executed in a Node.js context, let's stick with CJS modules for this file.

So I think you'd just need to replace `export default` with `module.exports = `

```suggestion
module.exports = class GitUtils {
```

...but be sure to retest :) You'll probably have to change the way you import it into the unit tests. Either `import * as GitUtils from '../lib/GitUtils';` or `const GitUtils = require('../lib/GitUtils');` would work I think.

Reply:
👍 I changed it to use `module.export` and used `require()` everywhere for consistency.

Reply:
The unit test is working as expected.

```
tugba ~/Documents/GitHub/expensify-common (tugbadogan-getPullRequestsMergedBetween)$ npm test GitUtils-test

> expensify-common@1.0.0 test /Users/tugba/Documents/GitHub/expensify-common
> jest "GitUtils-test"

 PASS  __tests__/GitUtils-test.js
  GitUtils
    getPullRequestsMergedBetween
      ✓ Test getPullRequestsMergedBetween by mocking exec (3 ms)
      ✓ Test getPullRequestsMergedBetween by mocking exec
      ✓ Test getPullRequestsMergedBetween by mocking exec (1 ms)

Test Suites: 1 passed, 1 total
Tests:       3 passed, 3 total
Snapshots:   0 total
Time:        1.356 s
Ran all test suites matching /GitUtils-test/i.
```